index.html

```diff
@@ -123,6 +123,71 @@
+            <!-- password visualization -->
+            <svg class="Earnings__right-open js-eye-open" width="24" height="18" viewBox="0 0 24 18" fill="none" xmlns="http://www.w3.org/2000/svg" aria-label="password visualization">
+              <circle cx="12" cy="9" r="3" stroke="#98A2B3" stroke-width="2" />
+              <path d="M1 8.89277C6 -0.607207 17 -2.60723 23 8.89277C16.5 19.8928 7 19.3928 1 8.89277Z" stroke="#98A2B3" stroke-width="2" stroke-linejoin="round" />
+            </svg>
+            <!-- hide password -->
+            <svg class="Earnings__right-closed js-eye-closed" width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg" aria-label="hide password">
+              <g clip-path="url(#clip0_16_1806)">
+                <path d="M9.9 4.24C10.5883 4.07888 11.2931 3.99834 12 4C19 4 23 12 23 12C22.393 13.1356 21.6691 14.2047 20.84 15.19M14.12 14.12C13.8454 14.4147 13.5141 14.6512 13.1462 14.8151C12.7782 14.9791 12.3809 15.0673 11.9781 15.0744C11.5753 15.0815 11.1752 15.0074 10.8016 14.8565C10.4281 14.7056 10.0887 14.481 9.80385 14.1962C9.51897 13.9113 9.29439 13.5719 9.14351 13.1984C8.99262 12.8248 8.91853 12.4247 8.92563 12.0219C8.93274 11.6191 9.02091 11.2218 9.18488 10.8538C9.34884 10.4859 9.58525 10.1546 9.88 9.88M1 1L23 23M17.94 17.94C16.2306 19.243 14.1491 19.9649 12 20C5 20 1 12 1 12C2.24389 9.6819 3.96914 7.65661 6.06 6.06L17.94 17.94Z" stroke="#98A2B3" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+              </g>
+              <defs>
+                <clipPath id="clip0_16_1806">
+                  <rect width="24" height="24" fill="white"/>
+                </clipPath>
+              </defs>
+            </svg>
+          </button>
+        </div>
+        </div>
+        <div class="Earnings__right-contrary">
+          <p class="Earnings__right-policy" aria-label="about ember's policies when joining ember">By creating an account, you agree to our 
+            <a class="Earnings__privacy-policy" href="#" aria-label="here you can view our privacy policy">Privacy Policy</a>, and <a class="Earnings__privacy-policy" href="#" aria-label="here you can consult the Electronic Communication Policy">Electronic Communication Policy</a>. </p>
+          <button type="submit" class="Earnings__right-buttonn" aria-label="click to enter ember">Sign Up</button>
+          <p class="Earnings__right-error" id="js-error-message" aria-label="error when entering">Please fill in all fields.</p>
+          <p class="Earnings__right-success" id="js-success-message" aria-label="successfully registered">You have successfully registered!</p>
+        </div>
+      </form>
+      <dl class="Earnings__right-info">
+        <dd class="Earnings__right-privacy">Privacy Policy</dd>
+        <dd class="Earnings__right-number">2023 © Lucas Marques</dd>
+      </dl>
```

Comment:
No creo que la definicion de `privacy policy` sea `lucas marques` 😄 .
```suggestion
      <div class="Earnings__right-info">
        <p class="Earnings__right-privacy">Privacy Policy</p>
        <p class="Earnings__right-number">2023 © Lucas Marques</p>
      </div>
```